exercism/hamming/hamming.src.js

```diff
@@ -0,0 +1,17 @@
+var Hamming = function() {};
+
+Hamming.prototype.compute =  function (a, b) {
```

Comment:
I'd also like to suggest some changes to your logic. 

Suggestion 1) try to avoid wrapping your entire function in an `if`/`else`. In this case you can use a "guard clause" 

like this:

```js
  if (dnaA.length !== dnaB.length) 
    throw new Error('DNA strands must be of equal length.');
  …
```

A guard clause either returns or throws so no else if necessary because if the condition is true we leave the function right now. 

Suggestion 2) use clear variable names. 

- `numberOfDifferences` is better then `diff`.
- `dnaA`, `dnaB` are better then `a` and `b`


All together: 

```js
Hamming.prototype.compute =  function (dnaA, dnaB) {
  if (dnaA.length !== dnaB.length) 
    throw new Error('DNA strands must be of equal length.');

  var numberOfDifferences = 0
  for (var i = 0; i <= dnaA.length; i++) {
    if (dnaA[i] != dnaB[i]) numberOfDifferences++;
  };

  return numberOfDifferences;
};
```